deeptools/alignmentSieve.py

```diff
@@ -7,15 +7,19 @@
 from deeptools import parserCommon
 from deeptools.bamHandler import openBam
 from deeptools.mapReduce import mapReduce
-from deeptools._version import __version__
+try:  # keep python 3.7 support.
```

Comment:
another reason why we should keep testing the lowest supported python version imho

Reply:
in addition to the lastest and fastest python version :)